panic: runtime error: invalid memory address or nil pointer dereference

$ git clone https://github.com/99designs/aws-vault.git
$ docker pull golang:1.9.2
$ cd  aws-vault
$ docker build . -t aws-vault
```
Sending build context to Docker daemon  10.48MB
Step 1/9 : FROM golang:1.9.2 AS build-env
 ---> 138bd936fa29
Step 2/9 : ENV CGO_ENABLED 0 GOOS linux GOARCH amd64
 ---> Using cache
 ---> c15882ebafbd
Step 3/9 : WORKDIR /go/src/github.com/99designs/aws-vault
 ---> Using cache
 ---> f02df53a573c
Step 4/9 : ADD . /go/src/github.com/99designs/aws-vault
 ---> Using cache
 ---> 71a988449203
Step 5/9 : RUN go build -a -tags netgo -ldflags '-w' -o /bin/aws-vault
 ---> Using cache
 ---> 733d0d546141
Step 6/9 : FROM alpine
 ---> e21c333399e0
Step 7/9 : COPY --from=build-env /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 ---> Using cache
 ---> 7f806162d349
Step 8/9 : COPY --from=build-env /bin/aws-vault /aws-vault
 ---> Using cache
 ---> d51fbe9b4871
Step 9/9 : ENTRYPOINT /aws-vault
 ---> Using cache
 ---> 68ecda91bf6e
Successfully built 68ecda91bf6e
Successfully tagged aws-vault:latest
```
$ docker run --rm -it aws-vault list
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x87f4fb]

goroutine 1 [running]:
github.com/99designs/aws-vault/cli.LsCommand(0xc4200b2690, 0x0, 0x0, 0x0)
	/go/src/github.com/99designs/aws-vault/cli/ls.go:59 +0x8b
github.com/99designs/aws-vault/cli.ConfigureListCommand.func1(0xc420108a20, 0x40ef63, 0x91fc40)
	/go/src/github.com/99designs/aws-vault/cli/ls.go:38 +0x7c
github.com/99designs/aws-vault/vendor/gopkg.in/alecthomas/kingpin%2ev2.(*actionMixin).applyActions(0xc4200b0198, 0xc420108a20, 0x0, 0x0)
	/go/src/github.com/99designs/aws-vault/vendor/gopkg.in/alecthomas/kingpin.v2/actions.go:28 +0x6d
github.com/99designs/aws-vault/vendor/gopkg.in/alecthomas/kingpin%2ev2.(*Application).applyActions(0xc4200b2690, 0xc420108a20, 0x0, 0x0)
	/go/src/github.com/99designs/aws-vault/vendor/gopkg.in/alecthomas/kingpin.v2/app.go:554 +0xe1
github.com/99designs/aws-vault/vendor/gopkg.in/alecthomas/kingpin%2ev2.(*Application).execute(0xc4200b2690, 0xc420108a20, 0xc4200590f0, 0x1, 0x1, 0x0, 0x0, 0xc4200f0b00, 0xc420035ee0)
	/go/src/github.com/99designs/aws-vault/vendor/gopkg.in/alecthomas/kingpin.v2/app.go:390 +0x8f
github.com/99designs/aws-vault/vendor/gopkg.in/alecthomas/kingpin%2ev2.(*Application).Parse(0xc4200b2690, 0xc42006a010, 0x1, 0x1, 0xc4200b2690, 0x4046a4, 0xc420056058, 0x0)
	/go/src/github.com/99designs/aws-vault/vendor/gopkg.in/alecthomas/kingpin.v2/app.go:222 +0x1f2
main.run(0xc42006a010, 0x1, 0x1, 0x9cd8e0)
	/go/src/github.com/99designs/aws-vault/main.go:36 +0x182
main.main()
	/go/src/github.com/99designs/aws-vault/main.go:14 +0x6f
```

--- a/cli/global.go
+++ b/cli/global.go
@@ -64,7 +64,6 @@ func ConfigureGlobals(app *kingpin.Application) {
 			if GlobalFlags.Backend != "" {
 				allowedBackends = append(allowedBackends, keyring.BackendType(GlobalFlags.Backend))
 			}
-
 			keyringImpl, err = keyring.Open(keyring.Config{
 				ServiceName:      "aws-vault",
 				AllowedBackends:  allowedBackends,
@@ -74,6 +73,9 @@ func ConfigureGlobals(app *kingpin.Application) {
 				KWalletAppID:     "aws-vault",
 				KWalletFolder:    "aws-vault",
 			})
+			if err != nil {
+				return err
+			}
 		}
 		if awsConfig == nil {
 			awsConfig, err = vault.LoadConfigFromEnv()

--- a/docker-compose.dev.yml
+++ b/docker-compose.dev.yml
@@ -1,0 +1,13 @@
+version: '2.1'
+
+services:
+  aws-vault:
+    image: golang:1.9.2
+    environment:
+      - CGO_ENABLED=0
+    volumes:
+      - .:/go/src/github.com/99designs/aws-vault
+      - /root/.aws
+      - /root/.awsvault
+    working_dir: /go/src/github.com/99designs/aws-vault
+    entrypoint: go run main.go

